extraConfigDpuTenant.py

```diff
@@ -92,6 +92,24 @@ def run(self, cfg):
         print("Creating network attachement definition")
         tclient.oc("create -f manifests/tenant/nad.yaml")
 
+        # Workaround for PCI realloc issue.
+        need_pci_realloc = False
+        for e in self._cc["workers"]:
+            rh = host.RemoteHost(tclient.get_ip(e['name']))
```

Comment:
`get_ip` might return None. Should never happen so error out in that case

Reply:
Got it.